.goreleaser.yaml

```diff
@@ -63,51 +62,50 @@ builds:
       - goos: linux
         goarch: arm64
     prebuilt:
-      path: '{{ .Env.GITHUB_WORKSPACE }}/{{ .Os }}_{{ .Arch }}{{ with .Amd64 }}_{{ . }}{{ end }}/lotus-worker'
-
+      path: "{{ .Env.GITHUB_WORKSPACE }}/{{ .Os }}_{{ .Arch }}{{ with .Amd64 }}_{{ . }}{{ end }}/lotus-worker"
+    skip: true
 archives:
   - id: primary
     format: tar.gz
     wrap_in_directory: true
-    name_template: "{{ .ProjectName }}_v{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+    name_template: "{{ .Env.PROJECT_NAME }}_v{{ .Env.VERSION }}_{{ .Os }}_{{ .Arch }}"
     files:
       # this is a dumb but required hack so it doesn't include the default files
       # https://github.com/goreleaser/goreleaser/issues/602
       - _n_o_n_e_*
-
 release:
   github:
     owner: filecoin-project
     name: lotus
-  prerelease: auto
-  name_template: "v{{.Version}}"
-
+  draft: true
+  header: "{{ .Env.HEADER }}"
+  make_latest: "{{ .Env.IS_LATEST }}"
+  name_template: "{{ .Env.TAG }}"
+  prerelease: "{{ .Env.IS_PRERELEASE }}"
+  replace_existing_artifacts: true
+  replace_existing_draft: false
```

Comment:
should we set this to `true` so we don't end up with multiple drafts if re-pushing the same branch?